components/common/NotificationBox/useNotification.ts

```diff
@@ -43,14 +45,17 @@ export const useNotification = (account: string) => {
   const { data: collectionData } = useGraphql({
     queryName: 'collectionByAccount',
     variables: {
-      account,
+      account: accountId.value,
     },
   })
-
-  const { data: eventData } = useGraphql({
+  const { data: eventData, loading } = useGraphql({
+    queryPrefix:
+      urlPrefix.value === 'bsx' || urlPrefix.value === 'snek'
```

Comment:
@roiLeo @preschian 
isn't it time we update the prefix resolver to avoid having to do these kind of hacks?

Reply:
point 'rmrk' to subsquid/general?